cmd/argocd/commands/admin/redis_initial_password.go

```diff
@@ -74,14 +79,14 @@ func NewRedisInitialPasswordCommand() *cobra.Command {
 				errors.CheckError(err)
 			}
 
-			fmt.Println("Argo CD Redis secret state confirmed: secret name argocd-redis.")
-			secret, err = kubeClientset.CoreV1().Secrets(namespace).Get(context.Background(), redisInitialPasswordSecretName, v1.GetOptions{})
+			fmt.Printf("Argo CD Redis secret state confirmed: secret name %s.\n", redisInitialCredentials)
```

Comment:
This change is notable -- when the code was added in 13d5803e7587f59db85862a368f03383447a1953, the text in the string contained a burned in reference to the secret name instead of relying on the constant/variable.